BetterMD/__init__.py

```
@@ -15,14 +15,19 @@ def from_file(file):
 
     @staticmethod
-    def from_url(url):
+    def from_url(url:'str'):
         import requests as r
         text = r.get(url).text
 
         if text.startswith("<!DOCTYPE html>"):
             text = text[15:]
 
-        return Symbol.from_html(text)
+        ret = Symbol.from_html(text)
```

Ruff (0.8.2)

`Symbol` may be undefined, or defined from star imports

(F405)